templates/partners/executive-summit.html

```diff
@@ -6,8 +6,10 @@
 
 {% block meta_description %}Connect with partners from around the globe to learn about open source, from data centre to the cloud, to maximize your company sales.{% endblock %}
 
+{% set is_paper = True %}
```

Comment:
Another thing we could do here is instead of having `is_paper` in the [base_index.html](https://github.com/canonical/canonical.com/blob/eee7f73d656e65942f384138b8f74246d7f7221b/templates/base_index.html#L83) we should maybe change it to set a class. I am suggesting this because you guys have added the `is_grapphite` body class, so maybe setting a class instead of true or false is a better option moving on. https://github.com/canonical/ubuntu.com/blob/6b288d3e210f483fb382e8338ff694bed8a90054/templates/templates/base.html#L70

Reply:
Or even on step further would be to set all the pages with paper white background (as long as design is consistent with future designs) by default and adding classes to override

Reply:
Thanks @carkod, great shout about this. I went with the first option since we still have a few pages that are not supposed to be using the paper background